newrelic/structures_newrelic_nrql_alert_condition.go

```diff
@@ -709,7 +709,7 @@ func flattenNrql(nrql alerts.NrqlConditionQuery, configNrql map[string]interface
 	svRaw := configNrql["since_value"]
 
 	// Handle deprecated
-	if svRaw != nil && svRaw.(string) != "" {
+	if svRaw != nil && svRaw.(string) != "" && nrql.EvaluationOffset != nil {
 		evalOffset := nrql.EvaluationOffset
 		out["since_value"] = strconv.Itoa(*evalOffset)
```

Comment:
:memo: The source of the panic the user was seeing occurred here.  An evaluation offset can be nil was well because it is a deprecated field.